notebooks/15_flightplanning_tuto.ipynb

```diff
@@ -0,0 +1,1018 @@
+    "                )\n",
+    "                bearing = pt[i - 1][nc2 - j].initialBearingTo(pt[np2][nc2 - j])\n",
+    "                total_distance = pt[i - 1][nc2 - j].distanceTo(pt[np2][nc2 - j])\n",
+    "                pt[i][nc2 - j] = pt[i - 1][nc2 - j].destination(\n",
+    "                    total_distance / (np2 - i + 1), bearing\n",
+    "                )\n",
+    "                # second half (np2 to p1)\n",
+    "                bearing = pt[np2 + i - 1][nc2 + j].initialBearingTo(pt[np - 1][nc2 + j])\n",
+    "                total_distance = pt[np2 + i - 1][nc2 + j].distanceTo(\n",
+    "                    pt[np - 1][nc2 + j]\n",
+    "                )\n",
+    "                pt[np2 + i][nc2 + j] = pt[np2 + i - 1][nc2 + j].destination(\n",
+    "                    total_distance / (np2 - i + 1), bearing\n",
+    "                )\n",
+    "                bearing = pt[np2 + i - 1][nc2 - j].initialBearingTo(pt[np - 1][nc2 - j])\n",
+    "                total_distance = pt[np2 + i - 1][nc2 - j].distanceTo(\n",
+    "                    pt[np - 1][nc2 - j]\n",
+    "                )\n",
+    "                pt[np2 + i][nc2 - j] = pt[np2 + i - 1][nc2 - j].destination(\n",
+    "                    total_distance / (np2 - i + 1), bearing\n",
+    "                )\n",
+    "        return pt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Domain factory\n",
+    "\n",
+    "Because some of the solvers are parallel, one need to be able to create similar domains easily. This is done through a factory which is a simple `lambda` function.\n",
+    "For creating this function, we need to define the parameters of our flight.\n",
+    "\n",
+    "In our case, we will flight in `A380` from Charles de Gaulle airport, France (`LFPG`) to Changi airport, Singapore (`WSSS`). By default, the take off weight is 80% of the `MTOW`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain_factory = lambda: FlightPlanningDomain(\n",
+    "    \"LFPG\", \"WSSS\", \"A388\", wind_interpolator=wind_interpolator\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solving the problem\n",
+    "\n",
+    "Now that we have defined the domain we can proceed with finding a solution for it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding suitable solvers\n",
+    "The `scikit-decide` library includes a lot of solvers. We can use `match_solvers` function to show available solvers that fit the characteristics of the defined domain, according to the mixin classes used to define the [domain type](#Domain-type)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain = domain_factory()\n",
+    "\n",
+    "match_solvers(domain=domain)"
```

Comment:
unrelated to the notebook, why "match_solvers" is taking so long time ?

Reply:
That's the question I wanted to ask :-)

Reply:
@neo-alex @fteicht :)

Reply:
Actually, this `domain_factory()` which is taking  ~40s on my laptop (and only 1s for `match_solvers()`).
As they are in the same cell, you were mistaken...